proposals/lip-0027.md

```diff
@@ -483,25 +498,28 @@ A binary message is valid, with respect to the corresponding data and schema, if
 3. The binary message contains exactly one value for each property specified in the schema.
-    There are three exceptions to this rule:
-    1. Empty arrays do not appear in the binary message.
-    2. [Non-packed arrays](#array-of-string-object-and-bytes) may contain multiple values.
-    3. Non-required properties for which no value was provided do not appear in the binary message. In particular, the key is also NOT contained in the binary message.
-4. The binary message does not contain properties not present in the schema nor does it contain any additional bytes that cannot be decoded to key-value pairs.
+There are three exceptions to this rule:
+   1. Empty _Arrays_ do not appear in the binary message.
+   2. [Non-packed arrays](#array-of-string-object-and-bytes) may contain multiple values.
+   3. Non-required properties for which no value was provided do not appear in the binary message. In particular, the key is also NOT contained in the binary message.
+4.  The binary message does not contain properties not present in the schema.
 
 A few examples are available in the [Testing and Examples](#testing-and-examples) section, and more can be found in the [protobuf documentation](https://developers.google.com/protocol-buffers/docs/encoding).
 
+**Superseded in [LIP 0064](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0064.md)**: Point 3 becomes obsolete as there are no non-required properties anymore.
+
 ### Decoding
 
-Only [valid messages](#uniqueness_of_binary_messages) can be decoded. For invalid messages, decoding fails. When a valid binary message is parsed, the binary message is decoded according to the [proto2](https://developers.google.com/protocol-buffers/docs/encoding) specifications. In particular, the following rule regarding missing values will be applied:
+When a binary message is parsed, the binary message is decoded according to the [proto2](https://developers.google.com/protocol-buffers/docs/encoding) specifications. In particular, the following rules will be applied:
 
-Whenever a binary message does not contain a specific field, the corresponding field in the parsed object is set to its default value. The default value is type-specific:
-1. For strings, the default value is the empty string.
-2. For bytes, the default value is the empty bytes.
-3. For booleans, the default value is false.
-4. For numeric types, the default value is zero.
-5. For arrays, the default value is the empty array.
-6. For objects, the default value is an object in which each of the properties is set to its default value.
+1. Missing values: Whenever a binary message does not contain a specific field, the corresponding field in the parsed object is set to its default value. The default value is type-specific:
```

Comment:
I think here we want to have a nested list. The markdown output is just a regular list, however. Maybe the indentation needs to be changed.